engine/source/runtime/function/render/passes/main_camera_pass.cpp

```diff
@@ -27,6 +27,9 @@ namespace Pilot
     {
         RenderPass::initialize(nullptr);
 
+        const MainCameraPassInitInfo* _init_info = static_cast<const MainCameraPassInitInfo*>(init_info);
```

Comment:
local variable naming should be `const MainCameraPassInitInfo* init_info`

Reply:
the `init_info`  has been used in `const RenderPassInitInfo* init_info` as the parameter of `initialize` function. i referenced the  `initialize` function of `ColorGradingPass` which used this naming style.